src/colors/common.d.ts

```diff
@@ -1,17 +1,13 @@
 import { Color } from '..';
```

Comment:
Just noted ... they are actually just `string`s not `Color`s! :)

Reply:
Thanks again, it's pushed 😄